mozphab-macos.rst

```diff
@@ -4,7 +4,7 @@ macOS MozPhab Installation Guide
 
 MozPhab can be installed from PyPI
 
-This requires Git and Python 3.6 or higher with `pip`.
+This requires Git, Rust, and Python 3.6 or higher with `pip`.
```

Comment:
Hmm, that's unfortunate, theoretically you should be getting the `moz-phab` "wheel", which has pre-compiled native code. Let's clarify here that "Rust" being a requirement only affects some systems:

```suggestion
This requires Git and Python 3.6 or higher with `pip`.

.. note::

    There isn't a `moz-phab` wheel for all macOS systems, so you may need to install Rust so that `pip` can build `moz-phab` itself.
```

Reply:
Sounds good to me! If the user would get an exit error with Rust being missed, they will know that this is okay and can be fixed.

Thank you!